sync2jira/upstream_issue.py

```diff
@@ -358,6 +361,25 @@ def github_issues(upstream, config):
         if issue.get('milestone', None):
             issue['milestone'] = issue['milestone']['title']
 
+        if not issue.get('storypoints', None):
+            issue['storypoints'] = ''
+            graphqlurl = 'https://api.github.com/graphql'
+            orgname = "open-connectors"
+            reponame = "open-connectors"
+            issuenumber = issue['number']
+            github_project_fields = config['sync2jira']['map']['github']["/".join((orgname,reponame))]['github_project_fields']
+            for fieldname, values in github_project_fields.items():
+                ghfieldname = values[0]
+                query = """query MyQuery($orgname: String!, $reponame: String!, $ghfieldname: String!, $issuenumber: Int!) { repository(owner: $orgname, name: $reponame) { issue(number: $issuenumber) { title body projectItems(first: 1) { nodes { fieldValueByName(name: $ghfieldname) { ... on ProjectV2ItemFieldNumberValue { number } } } } } }}"""
+                variables = {"orgname": orgname, "reponame": reponame, "ghfieldname": ghfieldname, "issuenumber": issuenumber}
```

Comment:
`query` should probably be defined statically.  And, unless you're going to use line breaks and pretty formatting, you should use single-double-quotes rather than triple-double-quotes.

Also, since only `ghfieldname` changes inside the loop, consider moving the definition of `variables` outside the loop (possibly inlining and removing the local variables, `orgname`, `reponame`, and `issuenumber`), and then replacing line 374 with something like
```python
                variables["ghfieldname"] = ghfieldname
```

Reply:
Done. 46248b8bbea26b8508001cedd23126f6719599b9